### ISSUE TITLE
Deploy artifacts of Maven Central

### ISSUE THREAD

**Issue by hosuaby** [OpenFeign/feign-vertx#15]
Hello @kdavisk6 

I am creator and maintainer of this repository. Until present moment, all artifacts from this projet are deployed on JCenter. From 1 may 2021 Bintray will be disabled as this article states https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

So, I need to find the way to migrate artefacts of this project to Maven Central. But i have not right credentials for `feign` namespace.

Can you please help me to configure build pipeline to push artifacts to Maven Central ?

**Comment by kdavisk6** [comment]
I wanted to let you know that I saw this and I have some documentation for you that can help. I’ll provide it here once I’m at my workstation.

**Comment by kdavisk6** [comment]
Thanks for your patience.

1.  you need access to the group on Maven Central.  Create a new account the Sontatype JIRA and add a comment to this issue requesting access: https://issues.sonatype.org/browse/OSSRH-23228.  You can see my request there, just copy and paste the request.  It will take about a day.

2.  Once you have access, verify by logging into the OSSRH Nexus: https://oss.sonatype.org/ with the same credentials from JIRA.  Click on staging profiles and you should see the `io.github.openfeign` group.  

3.  Create an upload a gpg signature for artifact signing.  Sonatype does not allow unsigned artifacts.  They have great instructions here: https://central.sonatype.org/pages/working-with-pgp-signatures.html

4.  Configure your maven build to sign the artifacts and release: https://central.sonatype.org/pages/apache-maven.html.  You can also look at the main POM on `feign` for how we have it configured as well.  If you follow the directions here and want to do a dry run, change the `<autoReleaseAfterClose>true</autoReleaseAfterClose>` to be `false`

At this point you are ready to go.    I see that you are still using Travis.  That's gone now too.  We moved the main project to CircleCI.  Feel free to use our `.circleci/config.yml` for inspiration.  We have unlimited time on CircleCi, just limited parallel builds.

**Comment by hosuaby** [comment]
Thanks you @kdavisk6.

I made a request. Can you please leave a comment on this Jira to accept my request.

**Comment by hosuaby** [comment]
Hello, @kdavisk6 
Can you please check if I have administrator rights on this repository? I want to add some secrets to repository but I can button "Settings".

**Comment by kdavisk6** [comment]
Are you talking about in Circle?

**Comment by hosuaby** [comment]
Yes. I need to define username, password and gpg passwords as secrets in repository.

**Comment by kdavisk6** [comment]
Circle manages that through GitHub. I’ll need to add you to the organization here first. Once that’s done you will have access to org settings and secrets.

if you don’t want to use the Sonatype settings we have for the main project, just create a new Conext with your settings

**Comment by hosuaby** [comment]
@kdavisk6 I try to set up project on CircleCi. I logged in with organization OpenFeign. I can not enable build for the project. It tells me "Contact repo admin to set up".

**Comment by kdavisk6** [comment]
I just made you admin on the repository. Not sure why you weren’t in the first place. I also enabled the project in Circle. Let me know how it goes.

**Comment by hosuaby** [comment]
Hello, @kdavisk6 !

I successfully migrated all existing artifacts of Maven Central and created a new CI pipeline with GitHub Actions (I a more familiar with GH actions that CircleCI).

I thank you for your help!
I have last one demand. Can you please desible build on CircleCI. Because now on each build I receive email about the failure.
